project/AppDependencies.scala

```diff
@@ -31,6 +31,10 @@ object AppDependencies {
     "org.scalacheck"         %% "scalacheck"               % "1.14.0"         % scope,
     "uk.gov.hmrc"            %% "bootstrap-test-play-27"   % "2.25.0"         % scope,
     "uk.gov.hmrc"            %% "domain"                   % "5.11.0-play-27" % scope,
-    "com.vladsch.flexmark"    % "flexmark-all"             % "0.35.10"        % scope
+    "com.vladsch.flexmark"    % "flexmark-all"             % "0.35.10"        % scope,
+    "uk.gov.hmrc"            %% "webdriver-factory"        % "0.18.0"         % scope,
+    "org.scalatestplus"      %% "selenium-3-141"           % "3.2.0.0"        % scope,
+    "uk.gov.hmrc"            %% "zap-automation"           % "2.9.0"          % scope,
```

Comment:
We shouldn't need zap-automation for the latest version of webdriver-factory as it's bundled. see: https://github.com/hmrc/dast-config-manager/blob/master/README.md